great_expectations/validator/validator.py

```diff
@@ -601,208 +594,6 @@ def inst_expectation(*args: dict, **kwargs):  # noqa: PLR0912
 
         return inst_expectation
 
-    def _build_expectation_configuration(  # noqa: PLR0913
```

Comment:
Hello again reviewer! Look at all this red! Double check me, but here's the deal:
* We removed auto and profiler_config from expectations in a previous PR
* That means none of the logic in `inst_rule_based_profiler` was relevant any longer
* That meant that `_build_expectation_configuration` became just a wrapper around calling the `ExpectationConfiguration` constructor, which meant it could be removed

Reply:
love it!